eslint.config.js

```diff
@@ -0,0 +1,15 @@
+import eslintConfigPrettier from 'eslint-config-prettier'
```

Comment:
This config is slightly different than the ADC starter. 

https://prettier.io/docs/en/integrating-with-linters.html

It seems like some of the eslint / prettier plugins aren't their top recommendation anymore.